lib/checks/aria/aria-required-attr-evaluate.js

```diff
@@ -57,7 +56,26 @@ function ariaRequiredAttrEvaluate(node, options = {}, virtualNode) {
   }
 
   if (missing.length) {
-    this.data(missing);
+    const isCombobox = role === 'combobox';
+    const messageKey = null;
+    if (isCombobox) {
```

Comment:
We probably don't need any of the new messages and could just leave it to saying we require `aria-controls` by default. If the user uses `aria-owns` instead we pass the check.